tests/constants/mod.rs

```diff
@@ -1,7 +1,21 @@
+use assert_fs::fixture::FileWriteStr;
+
 pub const ZKEVM_LLVM: &str = "zkevm-llvm";
 pub const PACKAGE_VERSION: &str = env!("CARGO_PKG_VERSION");
 pub const ERA_LLVM_REPO_URL: &str = "https://github.com/matter-labs/era-compiler-llvm";
 pub const ERA_LLVM_REPO_TEST_BRANCH: &str = "v1.3.10";
 pub const ERA_LLVM_REPO_TEST_REF: &str = "11de9aa24";
 pub const ERA_LLVM_REPO_TEST_SHA_INVALID: &str = "12345abcd";
 pub const LLVM_LOCK_FILE: &str = "LLVM.lock";
+
+/// Creates a temporary lock file for testing.
```

Comment:
`constants` is only for constants :)
Please make it a method of `Lock`.

Reply:
I found in the documentation that the preferred approach for the integration tests directory is to create a `common/mod.rs` module and put all common code and constants in it. This is described [here](https://doc.rust-lang.org/book/ch11-03-test-organization.html#:~:text=This%20is%20the,the%20test%20output.). The main idea of that is to ignore the common file for testing scenarios and not to print unnecessary verbose output for it.

Also, as we've discussed with you, I've decided to keep the function in the `tests` folder. To remove the inconsistency, `constants` is renamed to `common` which looks good to me and follows official Rust guidelines.